rpc/cache/service.proto

```diff
@@ -59,3 +60,7 @@ message MissingBlobsResponse {
   bool            missing_artifact = 1;
   repeated string missing_blob_ids = 2;
 }
+
+message DeleteBlobRequest {
+  string          blob_id = 1;
```

Comment:
I realized `blob_ids` may be better to reduce RPC calls.

Reply:
done